wire-tests-proto3-swift/src/main/swift/ContainsDuration.swift

```diff
@@ -5,6 +5,7 @@ import Wire
 
 public struct ContainsDuration {
 
+    @Defaulted(defaultValue: Duration())
```

Comment:
this seems to be a bug actually. Duration cannot be initialized as it has required fields....

Reply:
investigating...

Reply:
uh `messageType.fields` is empty for this proto...this is weird? I am updating the logic to handle this case...

Reply:
fixed by checking if `fields.isEmpty` but I do not love it, wonder why Wire wouldnt load those? `contains_duration.proto` has `import "google/protobuf/duration.proto";` ...